db-service/CHANGELOG.md

```diff
@@ -4,6 +4,14 @@
 - The format is based on [Keep a Changelog](http://keepachangelog.com/).
 - This project adheres to [Semantic Versioning](http://semver.org/).
 
+## [1.6.5](https://github.com/cap-js/cds-dbs/compare/db-service-v1.6.4...db-service-v1.6.5) (2024-03-06)
+
+
+### Fixed
+
+* deep delete for views without navigation ([#434](https://github.com/cap-js/cds-dbs/issues/434)) ([3ebc9c2](https://github.com/cap-js/cds-dbs/commit/3ebc9c29362045916c9060528729320c0eae4391))
```

Comment:
remove before release ([release-please does not do this for us](https://github.com/googleapis/release-please/issues/296))